crates/protocol/src/types/status_object.rs

```diff
@@ -0,0 +1,36 @@
+//! Minecraft's server status object
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct Version {
+    pub name: String,
+    pub protocol: crate::ProtocolVersion,
+}
+
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct SamplePlayer {
+    pub name: String,
+    pub id: uuid::Uuid,
+}
+
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct Players {
+    pub max: u32,
+    pub online: u32,
+    pub sample: Vec<SamplePlayer>,
+}
+
+#[derive(Debug, Clone, Deserialize, Serialize)]
+#[serde(rename = "camelCase")]
+pub struct StatusObject {
+    pub version: Version,
+    pub players: Players,
+    pub description: super::Chat,
```

Comment:
```suggestion
    pub description: Chat,
```

with

```
use crate::types::Chat;
```

otherwise this code is too location-specific without an obvious reason for it.

Reply:
Done